duradmin/src/main/webapp/WEB-INF/jsp/spaces/spaces-manager.jsp

```diff
@@ -31,7 +31,7 @@
 
               <c:if test="${error != null}">
                   alert("<c:out value="${error}"/>");
-              </c:if>      
+              </c:if>
```

Comment:
This formatting change probably should be reverted or at least added as a separate formatting themed commit since it is not related to duracloud-1214 (as far as I can tell).